code/modules/clothing/neck/cloaks.dm

```diff
@@ -0,0 +1,57 @@
+//Cloaks. No, not THAT kind of cloak.
+
+/obj/item/clothing/neck/cloak
+	name = "gray cloak"
+	desc = "It's a cape that can be worn around your neck in a pretty dull color."
+	icon_state = "cloak"
+	w_class = WEIGHT_CLASS_SMALL
+	body_parts_covered = UPPER_TORSO|ARMS
```

Comment:
The standard for bitwise `OR` is `flag | flag`, so need some extra spacing here:
```suggestion
	body_parts_covered = UPPER_TORSO | ARMS
```